src/app/containers/Ad/MPU.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, act } from '@testing-library/react';
+import { StaticRouter } from 'react-router-dom';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import MPU from './MPU';
+
+const toggles = {
+  ads: {
+    enabled: true,
+  },
+};
+
+describe('MPU', () => {
+  it('should render correctly', async () => {
```

Comment:
```suggestion
  it('should render without gel margins at all breakpoints and gel padding at smaller breakpoints', async () => {
```
We see this a lot in our codebase but I think it's good to be explicit about what rendering correctly means for each component.

Reply:
Good shout, have updated it